network/config.go

```diff
@@ -48,6 +49,7 @@ func DefaultConfig() *Config {
 		ForcePrivateNetwork:  false,
 		DefaultPort:          0,
 		IsBootstrapper:       false,
+		PeerStorePath:        "peers.json",
```

Comment:
Don't we need to have absolute path?

Reply:
No, as I tested it will locate the file on `pactus/peers.json`.